cmd/shield/main.go

```diff
@@ -1485,11 +1486,23 @@ func main() {
 				return err
 			}
 
-			if err := RunJob(id, string(b)); err != nil {
+			var taskUUID string = ""
+			if taskUUID, err = RunJob(id, string(b)); err != nil {
 				return err
 			}
 
-			OK("Scheduled immediate run of job")
+			if taskUUID != "" && *opts.Raw {
+				RawJSON(map[string]interface{}{
+					"ok":        "Scheduled immediate run of job",
+					"task_uuid": taskUUID,
+				})
+			} else {
+				OK("Scheduled immediate run of job")
+				if taskUUID != "" {
+					ansi.Printf("To view task, type @B{shield show task %s}\n", taskUUID)
+				}
+			}
+
```

Comment:
Is there a case where *opts.Raw is true, but taskUUID is still an empty string (a case not caught by the error handling above?)  It kind of looks like that's a hole in this if/else block, which would lead `--raw` usage to return non-JSON.